### ISSUE TITLE
redux 2.0 breaking changes

### ISSUE THREAD

**Issue by e-schultz** [wbuchwalter/ng-redux#15]
Looking at the redux breaking changes, and the bc-1.0.0 branch, looks like compose was changed which may affect this line

```javascript
let finalCreateStore = _storeEnhancers ? compose(..._storeEnhancers, createStore) : createStore;
```

looks like you might just be able to change it to

```javascript
let finalCreateStore = _storeEnhancers ? compose(..._storeEnhancers)(createStore) : createStore;
```

**Comment by wbuchwalter** [comment]
:+1: 
I'm wondering if I should release that + #12 as `2.0.0` instead of `1.0.0` to follow Redux version for clarity.
This is what react-redux does.

**Comment by e-schultz** [comment]
Might not be a bad idea, but could see the need to diverge later on, for example if you need to introduce a breaking change at 2.0.0, should bump to 3 - but wouldn't want to hold off until redux was at 3.0

**Comment by wbuchwalter** [comment]
Yes, if we can't stay at the same version, I don't have a problem diverging, but let's do it while we can :)

**Comment by wbuchwalter** [comment]
Fixed on `bc-2.0.0` branch, will close when merged!
Thanks
